content/javascript/concepts/arrow-functions/arrow-functions.md

```diff
@@ -33,8 +28,14 @@ The syntax for an arrow function expression does not require the `function` keyw
 
 Arrow functions can be defined with zero or more arguments. They can also be defined on one or more lines.
 
-> **Note:** Arrow functions functions are limited and can't be used in all situations:
->
+There are several variations of arrow functions:
+- Arrow functions with a single parameter do not require `()` around the parameter list.
+- Arrow functions with a single expression (no curly brackets `{ }`) can use the concise function body which returns the result of the expression without the `return` keyword.
+
+## Limitations
+
+Arrow functions functions are limited and can't be used in all situations:
+
 > - They do not have their own bindings to [`this`](https://www.codecademy.com/resources/docs/javascript/this) or `super`, and should not be used as [methods](https://www.codecademy.com/resources/docs/javascript/methods).
 > - They cannot be used as [constructors](https://www.codecademy.com/resources/docs/javascript/constructors).
 > - They cannot use `yield`, within its body.
```

Comment:
```suggestion
Arrow functions can't be used in all situations dud to the following limitations:

- They do not have their own bindings to [`this`](https://www.codecademy.com/resources/docs/javascript/this) or `super`, and should not be used as [methods](https://www.codecademy.com/resources/docs/javascript/methods).
- They cannot be used as [constructors](https://www.codecademy.com/resources/docs/javascript/constructors).
- They cannot use `yield` within its body and create generators.
```

Reply:
Can we also remove the block quote and space from the line below? 😄

Reply:
Can you specify what space you want me to remove?